.jshintrc

```diff
@@ -0,0 +1,32 @@
+{
+	"predef": [
+		"setImmediate",
+		"Map",
+		"Set"
+	],
+
+	"bitwise": true,
+	"laxbreak": true,
+	"curly": true,
+	"eqeqeq": true,
+	"immed": true,
+	"latedef": "nofunc",
+	"newcap": true,
+	"noarg": true,
+	"noempty": true,
+	"nonew": true,
+	"regexp": false,
+	"undef": true,
+	"strict": true,
+	"trailing": true,
+
+	"smarttabs": true,
+	"multistr": true,
+
+	"node": true,
+
+	"nomen": false,
+	"loopfunc": true,
+        "esnext": true
+	//"onevar": true
```

Comment:
Minor / trivial: Perhaps remove this? It's [deprecated anyway](https://core.trac.wordpress.org/ticket/28236?cversion=0&cnum_hist=8).